bin/pivotal-slacker

```diff
@@ -87,11 +87,26 @@ end
 
 # Shared functionality for outputting lists of stories (e.g. for `mine`, `started`, etc.).
 # Handles menuing functionality if opts[:menu] is `true`.
-def output_story_list opts={:project => nil, :states => [], :owned_by => nil, :menu => false}
-  puts "Looking for stories owned by #{opts[:owned_by]} in states: #{opts[:states]}".color("#444444")
+def output_story_list opts={:project => nil, :states => [], :owned_by => false, :menu => false, :labels => nil}
+  if opts[:owned_by].nil?
+    msg = "Looking for unclaimed stories in states: #{opts[:states]}"
+    args = {:current_state => opts[:states]}
+    if not (opts[:labels].nil? or opts[:labels].empty?)
+      msg += " and labels: #{opts[:labels]}"
+      msg = msg.color("#444444")
+      args[:search] = "current_state:#{opts[:states].join(',')}"
+      for label in opts[:labels]
+        args[:search] += " label:#{label}"
+      end
+    end
+    puts msg
+    stories = opts[:project].stories.all(args).select { |s| s.owned_by == nil }.reverse
```

Comment:
Probably worth noting that I had to do the `select` because the PT API was returning invalid result (no results) when `no:owner` was included in the filter.